tests/kernel/fatal/message_capture/src/main.c

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2012-2014 Wind River Systems, Inc.
```

Comment:
If this is a new added file, please change this Copy right to:  "Copyright (c) 2020 Intel Corporation"

Reply:
This is really a problem that needs attention. Thanks.